### PR TITLE
[Nicole]fix some typos in 3 lyric files

### PR DESCRIPTION
**Pull request by andy5995**

### CODE REVIEW

File: database/N/Nicole/Butterfly/Butterfly [hunk 1]
```diff
@@ -37,7 +37,7 @@ and watched it with a smile
 
 Soon the quarreling was over
 There was peace throughout the land
-Making all the days of travelling
+Making all the days of traveling
 seem worth while
```
Comment (snshn) [inline database/N/Nicole/Butterfly/Butterfly:41]: I think "worth while" should be one word here as well

File: database/N/Nicole/A Little Peace/Butterfly [hunk 2]
```diff
@@ -37,7 +37,7 @@ and watched it with a smile
 
 Soon the quarreling was over
 There was peace throughout the land
-Making all the days of travelling
+Making all the days of traveling
 seem worth while
```
Comment (snshn) [inline database/N/Nicole/A Little Peace/Butterfly:41]: I think "worth while" should be one word

File: database/N/Nicole/Butterfly/Butterfly (hunk 1 above)
Comment (andy5995) [reply database/N/Nicole/Butterfly/Butterfly:41]: Ok, @snshn I'll make that change. Btw, you might like to check out option #6 on this page. It's a nice timesaver for reviewers when reviewing ;) https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request

Once that's used, it appears like this (3) https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes

Merry Christmas, btw :)

Comment (snshn) [reply database/N/Nicole/Butterfly/Butterfly]: Merry Xmas to you too, Andy!

**Review by snshn — APPROVED** [review]

**Pull request merged**